mol_dqn/plot/target_sas_results.csv

```diff
@@ -0,0 +1,63 @@
+,index,ckpt,target_sas,sas,original_sas
+0,0,200000,2.5,2.5464338547671215,2.503945678145536
```

Comment:
Please include the SMILES for the starting and ending compounds in this DataFrame.

Reply:
I don't have the SMILES for the ending compounds. If you think it is necessary, I can rerun all the evaluations but I will have to update the figures and statistics, as the results will not be the same.

Reply:
Yes, I think this is important to have. Please do rerun if it isn't too much trouble.

Reply:
Done.